backend/src/sync.js

```diff
@@ -33,11 +33,11 @@ async function saveBlocks(blocksInfo) {
             if (chunks === undefined) {
```

Comment:
This `if` block can be completely removed as nearcore has the chunks implemented long time ago and the testnet has them as well.